pytest_pyodide/_decorator_in_pyodide.py

```diff
@@ -0,0 +1,95 @@
+import ctypes
```

Comment:
Could you add some comments on what this file does, so that people can know that this if not used in host but in Pyodide

Reply:
See also #43.